installation/vs-code-integration/scaffoldings.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Telerik UI for .NET MAUI Code Scaffoldings
```

Comment:
I suggest using a title that starts with the keyword (Scaffold or scaffolding) and also sticking with the main form without adding "s". Also try to put the emphasis on the action, on the things you can achieve with the feature, e.g. you can scaffold projects or apps.
Alternatively, you can use "Scaffolding MAUI Apps".
```suggestion
title: Scaffolding MAUI Projects
```